airflow/providers/google/cloud/sensors/pubsub.py

```diff
@@ -170,22 +171,41 @@ def execute(self, context: Context) -> None:
                     subscription=self.subscription,
                     max_messages=self.max_messages,
                     ack_messages=self.ack_messages,
-                    messages_callback=self.messages_callback,
                     poke_interval=self.poke_interval,
                     gcp_conn_id=self.gcp_conn_id,
                     impersonation_chain=self.impersonation_chain,
                 ),
                 method_name="execute_complete",
             )
 
-    def execute_complete(self, context: dict[str, Any], event: dict[str, str | list[str]]) -> str | list[str]:
+    def execute_complete(self, context: Context, event: dict[str, str | list[str]]) -> Any:
```

Comment:
```suggestion
    def execute_complete(self, context: Context, event: dict[str, str | list[str]]) -> list[Any]:
```

Reply:
When the user provided messages_callback executes , we dont know the return type so i have added the Any?